services/user.py

```diff
@@ -0,0 +1,49 @@
+from db.models import User
+from django.contrib.auth import get_user_model
```

Comment:
It is worth importing according to PEP8 requirements.

Reply:
I'm not sure that I got your point about it. So can you look at this part again in the new commit?)